Lib/test/test_sqlite3/test_cli.py

```diff
@@ -200,5 +204,37 @@ def test_color(self):
             self.assertIn('\x1b[1;35mOperationalError (SQLITE_ERROR)\x1b[0m: '
                           '\x1b[35mnear "sel": syntax error\x1b[0m', err)
 
+
+@requires_subprocess()
+class Completer(unittest.TestCase):
```

Comment:
```suggestion
class CompletionTest(unittest.TestCase):
```

Reply:
I reverted to a similar naming as the original PR.